tests/unit/jobserver/test_middleware.py

```diff
@@ -137,6 +137,21 @@ class DummyView(DetailView):
     assert response.context_data["template_name"] == "my_template"
 
 
+def test_template_name_middleware_with_template_name_as_a_list_without_a_name(rf):
```

Comment:
thought: This is a really clunky/confusing name, but it's in line with the test above.

Reply:
This has gone now.